### ISSUE TITLE
Device attribute values blank in v0.6

### ISSUE THREAD

**Issue by fpagliughi** [Smithay/udev-rs#30]
In v0.6, when I iterate through the attributes of a device, I get a name for each, but the value for all of them is blank.

I discovered this in my own app, but it's the same in the example `list_devices`. Every device entry looks like:
```
  [properties]
    - "DEVPATH" "/devices/pci0000:00/0000:00:17.0/ata5/link5/ata_link/link5"
    - "SUBSYSTEM" "ata_link"
  [attributes]
    - "device" ""
    - "hw_sata_spd_limit" ""
    - "sata_spd" ""
    - "sata_spd_limit" ""
    - "subsystem" ""
    - "uevent" ""
```

For the moment I had to revert back to v0.5. That seems to work OK. I'll dig into it in more detail if I get a minute.

I'm running on Linux Mint 20 (Ubuntu 20.04).

**Comment by fpagliughi** [comment]
From the `lib-udev` library:

```
/**
 * udev_device_get_sysattr_list_entry:
 * @udev_device: udev device
 *
 * Retrieve the list of available sysattrs, with value being empty;
 * This just return all available sysfs attributes for a particular
 * device without reading their values.
 *
 * Returns: the first entry of the property list
 **/
_public_ struct udev_list_entry *udev_device_get_sysattr_list_entry(struct udev_device *udev_device) {
```

https://github.com/systemd/systemd/blob/9e3e592946668dc72ffba6f3598b244eaff6d1ae/src/libudev/libudev-device.c#L776

So, when iterating through the list of attributes, it appears that you still need to call `udev_device_get_sysattr_value()` for each one to get the value, in the way that v0.5 used to do.

**Comment by fpagliughi** [comment]
The fix is merged to master, so I will close the issue.

**Comment by Drakulix** [comment]
Thanks again and sorry for the long time this took. The fix was released as version 0.6.3.

**Comment by fpagliughi** [comment]
Thanks for getting the release out so quickly. It appears to have fixed my app.
